libs/langchain-community/src/llms/hf.ts

```diff
@@ -20,6 +22,11 @@ export interface HFInput {
    */
   maxTokens?: number;
 
+  /**
+   * The model will stop generating text when one of the strings in the list is generated.
+   */
+  stopSequences?: string[];
```

Comment:
The `stopSequences` input API has been deprecated in favor of the LCEL syntax: `model.bind({ stop: ... })`. Please verify that functionality doesn't already exist, and if not, could you implement it that way.